test/models/attachment_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AttachmentTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
**nitpick:** Can you remove this empty test or fill it in?

Reply:
Addressed in 009ade8a

Reply:
Thanks